src/main/java/io/appium/java_client/MobileBy.java

```diff
@@ -106,6 +106,17 @@ public static By androidDataMatcher(final String dataMatcherString) {
         return new ByAndroidDataMatcher(dataMatcherString);
     }
 
+    /**
+     * This locator strategy is only available in Espresso Driver mode.
+     * @param viewMatcherString is a valid class chain locator string.
```

Comment:
class chain?

Reply:
The sin of copy & paster...I got it from the dataMatcher. I'll add a new commit in real quick, for the dataMatcher too. My apologies...